### ISSUE TITLE
lvmctrld stuck due to `Global lock failed: error -210`

### ISSUE THREAD

**Issue by aleofreddi** [aleofreddi/csi-sanlock-lvm#21]
Found an instance stuck as follows:

```
E1111 17:31:06.529205       1 grpclogger.go:33] gRPC[1095848]: call /LvmCtrld/Vgs({}) returned error unexpected error: rc=5 stdout="  {\n      \"report\": [\n          {\n              \"vg\": [\n              ]\n          }\n      ]\n  }\n" stderr="  Global lock failed: error -210\n"
```

It seems `ENOLS` is -210

**Comment by aleofreddi** [comment]
It seems this happens when sanlock somehow fails to renew its leases.

To mitigate, I'll add wdmd and expose /dev/watchdog from the node (if not available, we'll auto load the `softdog` module).

**Comment by aleofreddi** [comment]
Fixed by 5f49a6b0a816c4b01867995c50ee4c02c56a8518.
